aequilibrae/parameters.yml

```diff
@@ -1,3 +1,7 @@
+entry_points:
```

Comment:
This gets shipped with AequilibraE and is used when we create new projects, so it would make sense making it somewhat useful.  An idea is to have something that will create a series of models KPIs:

* List of matrices with their totals
* Number of Links, Nodes and Zones
* List of result tables in the model